### DISCUSSION TITLE
FM01 Alternate Solution

### DISCUSSION THREAD

**Discussion by A1-exe** [Alic3C/Cyber-FastTrack-Spring-2021#23] [Ideas]
In conjunction to the current solution: 

Submitting the file at http://fotoforensics.com/ and looking at XMP metadata reveals the flag.
![image](https://user-images.githubusercontent.com/28128898/114107132-f8ffe100-9895-11eb-894b-d97d2f6f89a4.png)

**Comment by A1-exe** [comment]
I would just make a pull requests, but I can't at the moment

**Reply by RealJammy** [reply]
Why can't you make a PR?(Do leave screenshots/ quick recording)
